### ISSUE TITLE
Validated Versions Checkmark

### ISSUE THREAD

**Issue by afonsojramos** [spicetify/spicetify-marketplace#273]
### 📝 Provide a description of the new feature

Add two checkmarks for each theme, extension, and custom app to "officially" validate that the latest version of Spotify / Spicetify is supported by it. 

![image](https://user-images.githubusercontent.com/19473034/177434491-29746c6c-a2a2-41b4-86fd-eb60cfffd7c5.png)

This would work by adding two fields in each of the manifests. If people wanted to get the checks on their extensions then they need to update these values with the latest. This way forces a community-led effort of better detailing/finding when/what is not working in new versions. 

<img width="260" alt="image" src="https://user-images.githubusercontent.com/19473034/177434879-228b6cf6-e096-4561-a85a-db49d91cd250.png">

**Comment by CharlieS1103** [comment]
Wouldn't it be better to make the values a range? I.E "2.9.9-2.10.0" 
Then we just split at the hyphen

**Comment by afonsojramos** [comment]
Could be as well, yes, however, I'm not sure which one I prefer... In theory, we should always focus on the latest of everything as we've discussed again and again. Especially because you are talking about two different things. That is, it says that supports 2.9.9 and 1.1.88, but does it support 2.10.0 and 1.1.88? Or just 2.10.0 and 1.1.89? For this reason, I'd prefer to only guarantee the latest "pairing".

**Comment by CharlieS1103** [comment]
@afonsojramos That's certainly a good point, one artifact of this though might be that it will allow us to let users on older versions of spicetify download older versions of extensions and themes(if we can somehow link the latest verified functional state in the manifest to the respective commit in the repository. )

**Comment by afonsojramos** [comment]
Thinking in my head, that seems very complicated to do (fixing certain commit hashes, etc.), but it can be something to support in the future indeed.

**Comment by CharlieS1103** [comment]
@afonsojramos Yeah, we'd have to add an array of commit id's in themes and extensions manifests and allow users to select which commit to use is probs that way I would do it. Handling it automatically would just lead to more errors imo

**Comment by afonsojramos** [comment]
@CharlieS1103 that sounds good actually, but let's move that discussion to later. Once we close this issue we can touch again on this.
